### PR TITLE
fix: Update scaleway website documentation

### PR DESCRIPTION
**Pull request by abarbare**
Update Scaleway builder documentation.

### CODE REVIEW

File: website/pages/partials/builder/scaleway/Config-required.mdx [hunk 1]
```diff
@@ -18,7 +18,5 @@
   get the complete list of the accepted image UUID.
 
 - `commercial_type` (string) - The name of the server commercial type:
-  ARM64-128GB, ARM64-16GB, ARM64-2GB, ARM64-32GB, ARM64-4GB,
-  ARM64-64GB, ARM64-8GB, C1, C2L, C2M, C2S, START1-L,
-  START1-M, START1-S, START1-XS, X64-120GB, X64-15GB, X64-30GB,
-  X64-60GB
+  C1, C2L, C2M, C2S, DEV1-S, DEV1-M, DEV1-L, DEV1-XL,
+  GP1-XS, GP1-S, GP1-M, GP1-L, GP1-XL, RENDER-S
```
Comment (azr) [inline website/pages/partials/builder/scaleway/Config-required.mdx:22]: This file is extracted from the code; to change this part you have to change https://github.com/hashicorp/packer/blob/f8e31bff1218dc888b06039a99033a1f758c31be/builder/scaleway/config.go#L48-L53 and then run `make generate`

File: website/pages/docs/builders/scaleway.mdx [hunk 2]
```diff
@@ -43,52 +43,11 @@ can also be supplied to override the typical auto-generated key:
...
-
-- `boottype` (string) - The type of boot, can be either `local` or
-  `bootscript`. Default `local`
-
-- `bootscript` (string) - The id of an existing bootscript to use when
-  booting the server.
-
-- `remove_volume` (boolean) - Force Packer to delete volume associated with
-  the resulting snapshot after the build. Default `false`.
+@include 'builder/scaleway/Config-not-required.mdx'
```
Comment (azr) [inline website/pages/docs/builders/scaleway.mdx:50]: Nice !

File: website/pages/partials/builder/scaleway/Config-required.mdx (hunk 1 above)
Comment (abarbare) [reply website/pages/partials/builder/scaleway/Config-required.mdx:22]: Indeed, thanks for the tips, We'll do that :slightly_smiling_face:

**Comment by azr** [comment]
Ah, I forgot to add: don't forget to [read and sign the cla](https://cla.hashicorp.com/hashicorp/packer?pullRequest=10058) 🙂

**Comment by abarbare** [comment]
Just signed it and launched a force check on it, should be good in few minutes 🙂

**Comment by azr** [comment]
thanks !

**Pull request merged**